docs/development/maintainers.md

```diff
@@ -33,53 +32,60 @@ the latest release branch named `stable` (due to ReadTheDocs constraints).
      git shortlog -s LAST_TAG.. | cut -f2- | sort --ignore-case | tr '\n' ';' | sed 's/;/, /g;s/, $//' | fold -s
      ```
      where `LAST_TAG` is the tag for the last release.
-     Merge the PR.
 
-3. Assuming the upstream `stable` branch exists, rename it to a release branch
-   for the previous major version. For instance if last release was, `0.20.0`,
-   the corresponding release branch would be `0.20.X`,
+3. Make a PR with the updates from steps 1 and 2. Merge the PR.
+
+4. (Major release only.) Assuming the upstream `stable` branch exists,
```

Comment:
I was always confused about this. Our up-to-date branch is `main`, not `stable`. Shouldn't there be a command that merges main to stable?

Reply:
When we make a major release, say v0.21.0 we do
```
git branch v0.20.X stable
git branch -f stable v0.21.0
# remove branch protection on stable branch
git push -f upstream stable 
```